content/reference-data-api.md

```diff
@@ -1752,6 +1757,207 @@ Response:
 
 
 
+## Get Top Currencies ##
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routesV2/network/topCurrencies.js "Source")
+
+Returns the top currencies on the Ripple Consensus Ledger over a rolling 30 day window. The currencies are ordered from highest ranked to lowest, where the rank is determined by the volume and count of transactions and the number of unique counterparties. By default, returns the top currencies for the most recent date available. You can query historical data by date. _(New in [v2.1.0][])_
```

Comment:
A "rolling 30 day window" is a little ambiguous. What exactly does it mean here? That it will always use the last 30 days of data? Should clarify that. 

Also, perhaps rephrase the second sentence so that the definition of the ranking comes first. Depending on how you rephrase the first sentence, you might be able to wedge the sort order description there (i.e., "Returns the top currencies on the RCL for the last 30 days, sorted from highest to lowest ranking. The rank is determined by...").

Finally, it's unclear to me how the last sentence relates to the rest of the description.